projected_aperture_properties.py

```diff
@@ -664,40 +687,44 @@ def proj_veldisp_star(self) -> unyt.unyt_quantity:
     @lazy_property
     def ProjectedStellarInertiaTensor(self) -> unyt.unyt_array:
         """
-        Inertia tensor of the stars in projection.
+        Inertia tensor of the stellar mass distribution in projection.
+        Computed iteratively using an ellipse with area equal to that of a circle with radius
+        equal to the aperture radius. Only considers bound particles within the projected aperture.
         """
         if self.Mstar == 0:
             return None
-        return get_projected_inertia_tensor(
-            self.proj_mass_star, self.proj_pos_star, self.iproj
-        )
+        return get_projected_inertia_tensor(self.proj_mass_star, self.proj_pos_star, self.iproj, self.aperture_radius)
 
     @lazy_property
-    def ReducedProjectedStellarInertiaTensor(self):
+    def ProjectedStellarInertiaTensorReduced(self):
```

Comment:
Several of the new properties are missing `-> unyt.unyt_array:`

Reply:
Added in https://github.com/SWIFTSIM/SOAP/pull/82/commits/394ddd06eda98f6878b9906f5fb68193525b7c8a